bin/client/main.rs

```diff
@@ -246,8 +246,17 @@ async fn run() -> std::result::Result<(), ClientError> {
                 None => ensure_config_dir().await?.join("secret_keys.toml"),
             };
 
+            let match_type = match (sign_opts.label, sign_opts.label_matching) {
+                (Some(label), None) => Some(LabelMatch::FullMatch(label)),
+                (None, Some(label_matching)) => Some(LabelMatch::PartialMatch(label_matching)),
+                (None, None) => None,
+                _ => {
+                    unreachable!("both label and label-matching cannot be present at the same time")
```

Comment:
Interesting...did the compiler not think you'd exhausted all the enum options?

Reply:
`Some(), Some()` combination is still pending right, which is not possible as CLAP will throw error because `label` and `label-matching` conflict each other as we have set it to be

Reply:
Oh right. That makes total sense